lib/components/app/app-menu-item.tsx

```diff
@@ -68,7 +68,9 @@ export default class AppMenuItem extends Component<Props, State> {
   render(): JSX.Element {
     const { icon, id, onClick, subItems, text, ...otherProps } = this.props
     const { isExpanded } = this.state
-    const Element = otherProps.href ? 'a' : 'button'
+    const hasHref = !!otherProps.href
+    const isAbsolute = otherProps.href?.startsWith('http')
```

Comment:
If a url gets passed without an http should it still open a new browser tab?

Reply:
Then it would be a relative URL (like the route viewer or another resource), but that's a good question.

Reply:
Right now, relative URLs open in the same window per PR description.